InteractiveAutomationToolkit/Dialogs/Dialog.cs

```diff
@@ -607,6 +607,14 @@ public void Show(bool requireResponse = true)
 			}
 		}
 
+		/// <summary>
+		/// Hides the dialog window.
```

Comment:
Can you provide some more details? E.g. "Hides the dialog. This does not block any background logic from running. Use see cref="Show" (CANT FORMAT THIS IN GITHUB REVIEW) if you want to show the dialog again."